ceremonies/src/lib.rs

```diff
@@ -611,15 +608,31 @@ pub mod pallet {
 							meetup_index,
 							cid
 						);
-							return Err(<Error<T>>::VotesNotDependable.into())
+							(
+								Err(<Error<T>>::VotesNotDependable.into()),
+								MeetupResult::VotesNotDependable,
+							)
 						},
 						MeetupValidationError::IndexOutOfBounds => {
 							debug!(
 								target: LOG,
 								"index out of bounds for meetup {:?}, cid: {:?}", meetup_index, cid
 							);
-							return Err(<Error<T>>::MeetupValidationIndexOutOfBounds.into())
+							(
+								Err(<Error<T>>::MeetupValidationIndexOutOfBounds.into()),
+								MeetupResult::MeetupValidationIndexOutOfBounds,
+							)
 						},
+					};
+					// only mark issuance as complete in registering phase
+					// because in attesting phase there could be a failing early payout attempt
+					if current_phase == CeremonyPhaseType::Registering {
+						info!(target: LOG, "marking issuance as completed for failed meetup.");
+
+						<IssuedRewards<T>>::insert((cid, cindex), meetup_index, meetup_result);
+						return Ok(Pays::No.into())
```

Comment:
We should also return the error here. Otherwise, the user expects some rewards.

Reply:
If we return an error, the storage (IssuedRewards) will not be updated. We did these changes to fix it: https://github.com/encointer/pallets/issues/275
The "error" feedback is in the MeetupResult:
MeetupResult::VotesNotDependable or MeetupResult::MeetupValidationIndexOutOfBounds. Perhaps we can improve the name:  MeetupResult::VotesNotDependableError, MeetupResult::MeetupValidationIndexOutOfBoundsError ?

Reply:
Ah, sorry, I completely forgot that this was the cause of this PR. In that case, we should additionally add an event:

`MeetupEvaluated(MeetupResult)`